assets/global.js

```diff
@@ -1034,6 +1033,10 @@ class VariantSelects extends HTMLElement {
         const volumePricingSource = html.getElementById(
           `Volume-${this.dataset.originalSection ? this.dataset.originalSection : this.dataset.section}`
         );
+
+        const pricePerItemDestination = document.querySelector(`price-per-item .price-per-item`);
```

Comment:
We need to query by a unique id. Otherwise we'll run into issues with quick buy, because it's possible that multiple `price-per-item` elements are on the page.

See the examples in the same file that leverage `this.dataset.originalSection` and `this.dataset.section`